Xamarin.Forms.Platform.WinRT/EntryRenderer.cs

```diff
@@ -111,6 +111,7 @@ void TextBoxOnKeyUp(object sender, KeyRoutedEventArgs args)
 				return;
 
 			((IEntryController)Element).SendCompleted();
+			Control?.Unfocus();
```

Comment:
These Entries are used mostly for Data Entry Forms / User Name / Password boxes.
The Editor is used for Multi Line - Where Enter is handled differently

To me Enter means "Complete - I'm finished with this text box" So the logical thing to mean was to unfocus the box.